sdk/src/main/java/com/microsoft/hydralab/performance/InspectionStrategy.java

```diff
@@ -2,5 +2,27 @@
 // Licensed under the MIT License.
 package com.microsoft.hydralab.performance;
 
-public class InspectionStrategy {
+import java.io.Serializable;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class InspectionStrategy implements Serializable {
```

Comment:
How can I setup a perf test quickly if I was a new user of Hydra? I think the strategies are too complex for user to setup the test package

Reply:
Can add a default strategy if users enabled the perf test but no other config changed?

Reply:
> How can I setup a perf test quickly if I was a new user of Hydra? I think the strategies are too complex for user to setup the test package

Good question. To simplify the usage of performance testing, I suggest we remove the `when` filed. Which means: For Lifecycle strategy, we inspect the perf data during the whole lifecycle. And users don't need to choose when to inspect. cc @hydraxman

Reply:
Thanks for your comments. Let's create a new issue to track this: [Provide a default strategy if users enabled the perf test but no other config changed](https://github.com/microsoft/HydraLab/issues/289)